src/.eslitrc.json

```diff
@@ -0,0 +1,27 @@
+{
+  "env": {
+    "browser": true,
+    "es6": true,
+    "jest": true
```

Comment:
- There is a typographical error in the filename written above. ☝️
   Kindly change it form `.eslitrc.json` to `.eslintrc.json`. 😃